### PR TITLE
streamdeck: Add version 4.3.3.11845

### PR DESCRIPTION
**Pull request by TheZoker**
Add streamdeck:
https://www.elgato.com/en/gaming/downloads#sd-win

### CODE REVIEW

File: bucket/streamdeck.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "shortcuts": [
+        [
+            "StreamDeck.exe",
+            "StreamDeck"
+        ]
+    ],
+    "autoupdate": {
+        "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_$version.msi"
+    }
+}
```
Comment (Ash258) [inline bucket/streamdeck.json]:
```suggestion
}

```

File: bucket/streamdeck.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://www.elgato.com/en/gaming/downloads#sd-win",
+    "description": "Stream Deck lets you take control of your content and focus on what's important: your audience.",
+    "version": "4.3.3.11845",
+    "license": {
+        "identifier": "Freeware"
+    },
+    "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_4.3.3.11845.msi",
```
Comment (Ash258) [inline bucket/streamdeck.json]: > Requires Windows 10 64bit

```suggestion
    "architecture": {
        "64bit": {
            "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_4.3.3.11845.msi",
```

image

File: bucket/streamdeck.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883",
```
Comment (Ash258) [inline bucket/streamdeck.json]:
````suggestion
            "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883",
        }
    },

File: bucket/streamdeck.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "extract_dir": "Elgato\\StreamDeck",
+    "bin": "StreamDeck.exe",
+    "shortcuts": [
+        [
+            "StreamDeck.exe",
+            "StreamDeck"
+        ]
+    ],
+    "autoupdate": {
+        "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_$version.msi"
```
Comment (Ash258) [inline bucket/streamdeck.json]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_$version.msi"
            }
        }
```

**Comment by TheZoker** [comment]
Thanks for the feedback! Also implemented all changes here

File: bucket/streamdeck.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    },
+    "extract_dir": "Elgato\\StreamDeck",
+    "bin": "StreamDeck.exe",
+    "shortcuts": [
+        [
+            "StreamDeck.exe",
+            "StreamDeck"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/streamdeck.json]:
```suggestion
    ],
    "checkver": {
        "url": "https://gc-updates.elgato.com/windows/sd-update/final/app-version-check.json",
        "jsonpath": "$.Manual.Version"
    },
```

Comment (TheZoker) [reply bucket/streamdeck.json]: Done

File: bucket/streamdeck.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://www.elgato.com/en/gaming/downloads#sd-win",
+    "description": "Stream Deck lets you take control of your content and focus on what's important: your audience.",
+    "version": "4.3.3.11845",
+    "license": {
```
Comment (Ash258) [inline bucket/streamdeck.json]: I cannot find any URL providing URL.

```suggestion
    "license": "Freeware",
```

File: bucket/streamdeck.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://www.elgato.com/en/gaming/downloads#sd-win",
+    "description": "Stream Deck lets you take control of your content and focus on what's important: your audience.",
+    "version": "4.3.3.11845",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_4.3.3.11845.msi",
+           "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883"
```
Comment (Ash258) [inline bucket/streamdeck.json]: One last aestetic change, which is triggering me.

```suggestion
            "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883"
```

**Comment by TheZoker** [comment]
@Ash258 the check fails, because "the json is invalid", but I can't find any issues...

**Comment by Ash258** [comment]
Looks like issue of appveyor

A

B

**Comment by r15ch13** [comment]
I keep merging PRs an AppVeyor can't keep up. :grin:

**Comment by TheZoker** [comment]
So is this PR ready to merge? 😄

**Comment by r15ch13** [comment]
If @Ash258 doesn't find anything to complain about, yeah :smile:

**Comment by Ash258** [comment]
Done

A

**Pull request merged**